contrib/swww/template.py

```diff
@@ -1,15 +1,15 @@
 pkgname = "swww"
-pkgver = "0.8.1"
-pkgrel = 1
+pkgver = "0.8.2"
+pkgrel = 0
 build_style = "cargo"
 hostmakedepends = ["cargo"]
 makedepends = ["rust-std"]
 pkgdesc = "Solution to your Wayland Wallpaper Woes"
-maintainer = "Froggo <froggo8311@proton.me>"
+maintainer = "Nova <froggo8311@proton.me>"
 license = "GPL-3.0-only"
 url = "https://github.com/Horus645/swww"
```

Comment:
```suggestion
url = "https://github.com/LGFae/swww"
```

Reply:
Oh TIL

Reply:
it's the first line in the release notes